railties/lib/rails/generators/app_base.rb

```diff
@@ -316,7 +316,7 @@ def webpacker_gemfile_entry
         if options.dev? || options.edge?
           GemfileEntry.github "webpacker", "rails/webpacker", nil, "Use development version of Webpacker"
         else
-          GemfileEntry.version "webpacker", ">= 4.0.0.rc.3", "Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker"
+          GemfileEntry.version "webpacker", "~> 4.0.0", "Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker"
```

Comment:
Is there really a need to lock with patch version?

Reply:
At this point, no.
I thought that with the pace webpacker changes we might as well do this.
The code **would** require fewer version bumps if we don't specify the patch though. I'm open to either approach. Thanks for looking, @y-yagi!

Reply:
I can change it to "~> 4.0" if you think that's better.

Reply:
Basically, I think that should not be locked with patch unless there is a reasonable reason. It may cause leakage of version update. 
The update pace of webpacker is faster than Rails. We will not be able to pace with it. Also, if follow the semantic version, will not break by changing minor(of course, if can).

I think `~> 4.0` is better.